mw-webapp/src/logic/allWaysPage/AllWaysPage.tsx

```diff
@@ -195,6 +196,7 @@ export const AllWaysPage = () => {
         <Title
           level={HeadingLevel.h2}
           text={`${LanguageService.allWays.waysTable.leftTitle[language]} (${allWays.length})`}
+          dataCy={allWaysAccessIds.titles.allWays}
```

Comment:
waht do you think about a little bit shorter syntax `allWaysAccessIds` -> `allWaysId`

(we dont need to use 's' because this structure is not a list)

Reply:
Good point